Rephrase some explanations and add some missing things (#61)
CloudNetService/website: versioned_docs/version-3.4/commands/report.md

---

The `report` command is a useful command to gather information about the Cloud.

> This can be requested in the [support](../index.md#support), so that better help can be provided.

Reviewer: What does "this" refer to?

> This information can be requested in the [support](../index.md#support), so that better help can be provided.

Reviewer 2: I don't get this big change of the phrase... I guess this might be enough:

```suggestion
These information might be requested by our [support team](../index.md#support) to assist you with your support enquiry.
```

Author: I don't see the problem rephrasing it completely
